### PR TITLE
Added bandwidth history for usage on cloud

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: umbraco-cloud/set-up/project-settings/usage.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ On the Usage page, you will find an usage overview that displays your usage and
 
 ## Usage overview
 
-The usage overview shows the bandwidth usage of the project this month, the size of the media library and the number of custom domains added to the project. In this overview you will also find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
+The usage overview shows the bandwidth usage of the project this month, the size of the media library and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/usage.md:9]:
```suggestion
The overview shows the bandwidth of the project this month, the media library size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```

File: umbraco-cloud/set-up/project-settings/usage.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ On the Usage page, you will find an usage overview that displays your usage and
 
 ## Usage overview
 
-The usage overview shows the bandwidth usage of the project this month, the size of the media library and the number of custom domains added to the project. In this overview you will also find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
+The overview shows the bandwidth of the project this month, the media library size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/usage.md:9]:
```suggestion
The overview shows the bandwidth of the project for the current month, the media storage size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```

File: umbraco-cloud/set-up/project-settings/usage.md [hunk 3]
```diff
@@ -6,9 +6,11 @@ On the Usage page, you will find an usage overview that displays your usage and
...
 
-![Usage on Cloud](../images/Usage.png)
+In this overview you will also find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/usage.md:11]:
```suggestion
In this overview, you will find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
```

**Review by eshanrnh — COMMENTED** [review]
LGTM 🙌 Some minor suggestions..

**Comment by jonat123** [comment]
@eshanrnh Thanks, they all look good!

I've added them and I'll merge the PR

**Pull request merged**